### PR TITLE
clear authorization metadata

### PR DESCRIPTION
**Pull request by bourgeoa**

### CODE REVIEW

**Comment by bourgeoa** [comment]
https://github.com/linkeddata/rdflib.js/pull/512

File: src/login/login.ts [hunk 1]
```diff
@@ -506,6 +506,8 @@ export function renderSignInPopup (dom: HTMLDocument) {
 
   const loginToIssuer = async (issuerUri: string) => {
     try {
+      // clear authorization metadata from store
+      solidLogicSingleton.store.updater.flagAuthorizationMetadata() as any
```
Comment (angelo-v) [inline src/login/login.ts:510]: the metadata should be flagged as well on log out

Comment (bourgeoa) [reply src/login/login.ts:510]: It is done lines 711-712

Comment (bourgeoa) [reply src/login/login.ts:510]: https://github.com/SolidOS/solid-ui/blob/b361f8d2efc9a31785cd36aa5e8fc5829dcc9c63/src/login/login.ts#L712

Comment (angelo-v) [reply src/login/login.ts:510]: ah sorry, I thought it is login as well because github stripped away the context. all good

**Review by angelo-v — APPROVED** [review]

**Pull request merged**